pipelineUtilities.psm1

```diff
@@ -105,25 +128,18 @@ function Install-Dotnet {
         [string]$Channel = 'release'
     )
     try {
-        Find-Dotnet
-        return  # Simply return if we find dotnet SDk with the correct version
-    } catch { }
-    $obtainUrl = "https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain"
-    try {
-        $installScript = if ($IsWindows) { "dotnet-install.ps1" } else { "dotnet-install.sh" }
-        Invoke-WebRequest -Uri $obtainUrl/$installScript -OutFile $installScript
-        foreach ($majorMinorVersion in $DotnetSDKVersionRequirements.Keys) {
-            $version = "$majorMinorVersion.$($DotnetSDKVersionRequirements[$majorMinorVersion].DefaultPatch)"
-            Write-Host "Installing dotnet SDK version $version"
-            if ($IsWindows) {
-                & .\$installScript -InstallDir "$env:ProgramFiles/dotnet" -Channel $Channel -Version $Version
-            } else {
-                bash ./$installScript --install-dir /usr/share/dotnet -c $Channel -v $Version
-            }
+        $versionsToInstall = Find-DotnetVersionsToInstall
```

Comment:
Should we just add a global.json and the `UseDotNet` task?

Reply:
Following up on it later sounds good to me.

Reply:
https://github.com/Azure/azure-functions-core-tools/issues/3775